test/configuration_reader_test.exs

```diff
@@ -47,6 +47,15 @@ defmodule ExsodaTest.ConfigurationReader do
                                                                             value: %{"enabled" => true}},
                                                     %Configuration.Property{name: "fun",
                                                                             value: %{"enabled" => true}}],
-                                       type: "view_categories"}]
+                                       type: "view_categories"},
+                        %Configuration{id: 897,
+                                       name: "View categories",
```

Comment:
ah! they don't, and you're right, unrelated test that was failing (due to data source change)